teamC/JeongDayoung/app.js

```diff
@@ -0,0 +1,51 @@
+// require(“dotenv”).config();
+const http = require("http");
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+
+const app = express();
+
+app.use(express.json());
+app.use(morgan('dev'));
+
+dotenv.config();
+
+const myDataSource = new DataSource({
+  type : process.env.TYPEORM_CONNECTION,
+  host : process.env.TYPEORM_HOST,
+  port : process.env.TYPEORM_PORT,
+  username : process.env.TYPEORM_USERNAME,
+  password : process.env.TYPEORM_PASSWORD,
+  database : process.env.TYPEORM_DATABASE
+
+}) 
+
+myDataSource.initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  });
+
+app.get('/ping', cors(), function(req, res, next){
+  res.json({message : 'pong'})
+})
```

Comment:
arrow function을 사용해서 한 줄로 수정해주세요~
```suggestion
app.get('/ping', (req, res, next) => res.json({message : 'pong'})
```

Reply:
추가로 `cors()`는 line 13에 `app.use(cors())`로 선언해서 모든 엔드포인트 (/ping, /users 등)에 적용될 수 있도록 만들어주세요.